vars/parseStageInfo.groovy

```diff
@@ -195,6 +195,12 @@ def call(Map config = [:]) {
       }
     }
 
+    String target_branch = env.CHANGE_TARGET ? env.CHANGE_TARGET : env.BRANCH_NAME
+    if (target_branch == "release/1.2") {
+      echo "Updating tag to inclue daily_regression for release/1.2"
+      echo "Tag is ${tag}"
+      tag += ' daily_regression'
+    }
```

Comment:
Doing this here makes the addition absolute, even in cases where the user sets a Test-tag*.  That is not likely what we want.

It seems the goal here is to add `daily_regression` to PRs when normally it would have only been `pr`.  To achieve that, you want this addition to be done at https://github.com/daos-stack/pipeline-lib/blob/3f66aec1d9b61d96874ef6fcb6c70eceb18d096f/vars/parseStageInfo.groovy#L179.